src/Tickets/Emails/Email/Ticket.php

```diff
@@ -235,22 +295,40 @@ public function get_settings(): array {
 			],
 			static::$option_enabled => [
 				'type'                => 'toggle',
-				'label'               => esc_html__( 'Ticket Email ', 'event-tickets' ),
+				'label'               => esc_html__( 'Enabled', 'event-tickets' ),
 				'default'             => true,
 				'validation_type'     => 'boolean',
 			],
 			static::$option_subject => [
 				'type'                => 'text',
-				'label'               => esc_html__( 'Email subject', 'event-tickets' ),
+				'label'               => esc_html__( 'Subject', 'event-tickets' ),
 				'default'             => $this->get_default_subject(),
-				'placeholder'         => esc_html__( 'Your tickets to {event_name}', 'event-tickets' ),
+				'placeholder'         => $this->get_default_subject(),
+				'size'                => 'large',
+				'validation_callback' => 'is_string',
+			],
+			static::$option_subject_multiple => [
+				'type'                => 'text',
+				'label'               => esc_html__( 'Subject (multiple)', 'event-tickets' ),
```

Comment:
Any reason why you chose multiple over plural like it was in the ticket description?